database/fs/fs.go

```diff
@@ -0,0 +1,113 @@
+package fs
```

Comment:
I would keep this in the same package (in the end it's just another "database" implementation)